utensor_cgen/parser/_pbparser_impl.py

```diff
@@ -109,7 +110,8 @@ def _parse_graph_info(graph_def):
       if node.op in ["Const"]:
         for tensor_name, _, _ in output_tensor:
           tensor = graph.get_tensor_by_name(tensor_name)
```

Comment:
And remove `tensor = graph.get_tensor_by_name(...)` too.
It's not needed anymore.